pype/hooks/nukestudio/pre_launch_args.py

```diff
@@ -0,0 +1,27 @@
+import os
+from pype.lib import PreLaunchHook
+
+
+class NukeStudioLaunchArguments(PreLaunchHook):
+    order = 0
+    hosts = ["nukestudio"]
```

Comment:
this had been changed to `hiero` host already `nukestudio` want work anywhere as host.

Reply:
that apply to the name of folder